coderd/database/migrations/000389_telemetry_heartbeats.up.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE telemetry_heartbeats (
+    event_type TEXT NOT NULL CONSTRAINT telemetry_heartbeat_event_type_check CHECK (event_type IN ('aibridge_interceptions_snapshot')),
+    heartbeat_timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
```

Comment:
The table includes the term "heartbeat" already, and `created_at` aligns better with naming in other tables.

```suggestion
    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
```

Reply:
`created_at` doesn't make sense because the creation time of the row is not the same as the period the heartbeat was calculated. I wanted to call it just `timestamp` but `timestamp` is a type in postgres so we'd have to use double quotes wherever we referenced it. So I just decided to name it `heartbeat_timestamp` instead :(

Reply:
`period_ending_at`?